File: Problems/CCC/CCC20J2.java
package Problems.CCC;

import java.util.Scanner;

public class CCC20J2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int P = sc.nextInt();
        int N = sc.nextInt();
        int R = sc.nextInt();
        int day = 0;
        int infected = N;
        int oldInfected = N;
        while(infected <= P){
            day++;
            infected += oldInfected * R;
            oldInfected *= R;
        }


        System.out.println(day);
    }

}

File: Problems/CCC/CCC20J2.cpp
#include <iostream>

using namespace std;

int P, N, R;

int main()
{
    cin >> P >> N >> R;
    int infected = N;
    int old_infected = N;
    int day = 0;

    while(infected <= P){
        day ++;
        infected += old_infected * R;
        old_infected *= R;
    }
    cout << day << endl;
    return 0;
}